packages/extension/src/components/DaSidebar.vue

```diff
@@ -597,7 +597,7 @@ form.sidebar__element {
   bottom: 0;
   background: var(--theme-background-secondary);
   border-top: 1px solid var(--theme-separator);
-  z-index: 2;
+  z-index: 99999;
```

Comment:
The tooltip z-index is: `10000`
https://github.com/dailydotdev/daily-apps/blob/master/packages/components/src/components/DaTooltip.vue#L32
Maybe we should change the value to `10001`, so it will be more clear that is due to the tooltip.
What do you think?

Reply:
Sure